<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BibleImportDialog</class>
 <widget class="QDialog" name="BibleImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bible Registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/openlp-2.qrc">
    <normaloff>:/icon/openlp.org-icon-32.bmp</normaloff>:/icon/openlp.org-icon-32.bmp</iconset>
  </property>
  <widget class="QToolBox" name="ImportToolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>471</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="FileImportPage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>371</height>
     </rect>
    </property>
    <attribute name="label">
     <string>File Import Page</string>
    </attribute>
    <widget class="QGroupBox" name="OSISGroupBox">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>65</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>OSIS Bible</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="LocatioLabel">
        <property name="text">
         <string>File Location:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="OSISLocationEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="OsisFileButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../images/openlp-2.qrc">
          <normaloff>:/imports/import_load.png</normaloff>:/imports/import_load.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="CVSGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>451</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>CVS Bible</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="BooksLocationLabel">
        <property name="text">
         <string>Books Location:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="VerseLocationLabel">
        <property name="text">
         <string>Verse Location:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="VerseLocationEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="BooksLocationEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="BooksFileButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../images/openlp-2.qrc">
          <normaloff>:/imports/import_load.png</normaloff>:/imports/import_load.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="VersesFileButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../images/openlp-2.qrc">
          <normaloff>:/imports/import_load.png</normaloff>:/imports/import_load.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="BibleNameEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>280</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="BibleNameLabel">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>20</y>
       <width>98</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bible Name:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="WebBiblePage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>371</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Web Bible Import page</string>
    </attribute>
    <layout class="QVBoxLayout" name="WebBibleLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
     <item>
      <widget class="QGroupBox" name="OptionsGroupBox">
       <property name="title">
        <string>Download Options</string>
       </property>
       <widget class="QLabel" name="LocationLabel">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>27</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="LocationComboBox">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>27</y>
          <width>371</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Crosswalk</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="BibleLabel">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>54</y>
          <width>34</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Bible:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="BibleComboBox">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>54</y>
          <width>371</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NIV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KJV</string>
         </property>
        </item>
       </widget>
       <zorder>BibleComboBox</zorder>
       <zorder>LocationLabel</zorder>
       <zorder>BibleLabel</zorder>
       <zorder>LocationComboBox</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ProxyGroupBox">
       <property name="title">
        <string>Proxy Settings (Optional)</string>
       </property>
       <layout class="QFormLayout" name="ProxySettingsLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="horizontalSpacing">
         <number>8</number>
        </property>
        <property name="verticalSpacing">
         <number>8</number>
        </property>
        <property name="margin">
         <number>8</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="AddressLabel">
          <property name="text">
           <string>Proxy Address:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="AddressEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="UsernameLabel">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="UsernameEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="PasswordLabel">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="PasswordEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="LicenceDetailsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>435</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>123</height>
    </size>
   </property>
   <property name="title">
    <string>Licence Details</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>8</number>
    </property>
    <property name="margin">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="VersionNameLabel">
      <property name="text">
       <string>Version Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="VersionNameEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="CopyrightLabel">
      <property name="text">
       <string>Copyright:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="CopyrightEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="PermisionLabel">
      <property name="text">
       <string>Permission:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="PermisionEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="MessageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>451</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="ProgressGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>471</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>Import Progress</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QProgressBar" name="ProgressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>%p</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>680</y>
     <width>180</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="ImportButton">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BibleNameEdit</tabstop>
  <tabstop>OSISLocationEdit</tabstop>
  <tabstop>OsisFileButton</tabstop>
  <tabstop>BooksLocationEdit</tabstop>
  <tabstop>BooksFileButton</tabstop>
  <tabstop>VerseLocationEdit</tabstop>
  <tabstop>VersesFileButton</tabstop>
  <tabstop>LocationComboBox</tabstop>
  <tabstop>BibleComboBox</tabstop>
  <tabstop>AddressEdit</tabstop>
  <tabstop>UsernameEdit</tabstop>
  <tabstop>PasswordEdit</tabstop>
  <tabstop>VersionNameEdit</tabstop>
  <tabstop>CopyrightEdit</tabstop>
  <tabstop>PermisionEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../images/openlp-2.qrc"/>
 </resources>
 <connections/>
</ui>
